src/kibana/components/agg_response/geo_json/geo_json.js

```diff
@@ -33,7 +33,9 @@ define(function (require) {
           features: features,
           properties: {
             min: _.min(values),
-            max: _.max(values)
+            max: _.max(values),
+            zoom: _.deepGet(geoAgg, 'params.mapZoom'),
```

Comment:
FYI: `_.get` works the same as the new `_.get` method in lodash, which is a `_.deepGet` with a default.